Storybook/Pages/GameView/GameScene.swift

```diff
@@ -148,18 +160,18 @@ class GameScene: SKScene {
             spriteComponent.node.zPosition = 5
         }
         
-        entityManager.add(lionCub)
-        entityManager.add(lionMom)
-        entityManager.add(lionDad)
+        entityManager.add(lions)
+//        entityManager.add(lionMom)
+//        entityManager.add(lionDad)
     }
     
-    func makeLionTutorial() {
+    func makeCharacterTutorial(imageName: String?) {
         entityManager = EntityManager(scene: self)
         
-        let lionCub = Lion(imageName: "#1 Anak Singa")
-        if let spriteComponent = lionCub.component(ofType: SpriteComponent.self) {
-            spriteComponent.node.position = CGPoint(x: -spriteComponent.node.frame.midX/2-250, y: spriteComponent.node.frame.midY/2-85)
-            spriteComponent.node.size = CGSize(width: 550, height: 550)
+        let lions = Lion(imageName: imageName ?? "")
```

Comment:
Please implement this not just for 1 lion, you can see that it still implemented a hard code for the father & mother lion. @adamfiadi7007